app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -164,6 +172,19 @@ To teach in England, you will need a visa or immigration status allowing you to
 
 Learn more about [applying for the right visa for your teaching job](/non-uk-teachers/visas-for-non-uk-teachers). 
 
+<section class="inset-text purple">
```

Comment:
This looks fine on the front-end but I thought we normally defined these insets in the frontmatter and then called them in the body?

Paging @emmaachesongray for advice!